src/misc/lv_style.c

```diff
@@ -231,6 +231,7 @@ void lv_style_transition_dsc_init(lv_style_transition_dsc_t * tr, const lv_style
 lv_style_value_t lv_style_prop_get_default(lv_style_prop_t prop)
 {
     lv_style_value_t value;
+    value.ptr = 0;
```

Comment:
Fix for valgrind error: on AMD64 machine, if only writing to value.num, the high word is not set, resulting in conditional jump on undefined value.